modules/virt-about-conditions-and-events.adoc

```diff
@@ -7,6 +7,7 @@
 
 Diagnose DataVolume issues by examining the output of the `Conditions` and `Events` sections
 generated by the command:
+[source, terminal]
```

Comment:
While it appears to render correctly as-is, it's one of the OCP conventions to include a newline before all code blocks (after `command:`).

Reply:
Done.